File: codeChef/CPP files/chn15a.cpp
#include<iostream>

int main(){
    int test_cases;
    int N, K;
    int count;
    std::cin >> test_cases;
    while(test_cases--){
        std::cin >> N >> K;
        while(N--){
            int c;
            std::cin >> c;
            if((c+K)%7 == 0) count++;
        }
    }
    std::cout << count << std::endl;
    return 0;
}

File: codeChef/python files/chn15a.py
def mutateMinions(N, K, m):
    count = 0
    for i in [x+K for x in m ]:
        if i % 7 == 0:
            count += 1
    return count 


if __name__ == "__main__":
    test_cases = int(input())
    ans = list()
    for i in range(0, test_cases):
        N, K = map(int, input().split())
        m = list(map(int, input().split()))
        ans.append(str(mutateMinions(N, K, m)))

    print("\n".join(ans))